bin/build-hdfs

```diff
@@ -126,7 +126,20 @@ cp $PROJ_DIR/hdfs-scheduler/build/libs/*-uber.jar $BUILD_DIR/$DIST/lib
 cp $BUILD_CACHE_DIR/$EXECUTOR.tgz $BUILD_DIR/$DIST
 mkdir -p $BUILD_DIR/$DIST/etc/hadoop
 cp $PROJ_DIR/conf/*.xml $BUILD_DIR/$DIST/etc/hadoop
+
+echo "Copying required hadoop dependencies into $BUILD_DIR/$DIST for the scheduler"
+cp $BUILD_CACHE_DIR/$HADOOP_DIR/bin/* $BUILD_DIR/$DIST/bin
+cp $BUILD_CACHE_DIR/$HADOOP_DIR/etc/* $BUILD_DIR/$DIST/etc
+cp -R $BUILD_CACHE_DIR/$HADOOP_DIR/libexec $BUILD_DIR/$DIST
+mkdir -p $BUILD_DIR/$DIST/share/hadoop/common
+cp -R $BUILD_CACHE_DIR/$HADOOP_DIR/share/hadoop/common/hadoop-common-$HADOOP_VER.jar $BUILD_DIR/$DIST/share/hadoop/common
+cp -R $BUILD_CACHE_DIR/$HADOOP_DIR/share/hadoop/common/lib $BUILD_DIR/$DIST/share/hadoop/common
+mkdir -p $BUILD_DIR/$DIST/share/hadoop/hdfs
+cp -R $BUILD_CACHE_DIR/$HADOOP_DIR/share/hadoop/hdfs/hadoop-hdfs-$HADOOP_VER.jar $BUILD_DIR/$DIST/share/hadoop/hdfs
+cp -R $BUILD_CACHE_DIR/$HADOOP_DIR/share/hadoop/hdfs/lib $BUILD_DIR/$DIST/share/hadoop/hdfs
+cp -R $BUILD_CACHE_DIR/$HADOOP_DIR/share/hadoop/hdfs/webapps $BUILD_DIR/$DIST/share/hadoop/hdfs
```

Comment:
I don't think this is necessary.  Is this what was previously there?   We build a shadow "fat" jar for the scheduler.   That means we should be able to add a one line dependency in a gradle build file and all these dependencies would be in the fat jar with the need to have a bunch of copy commands here.